.github/workflows/makefile.yml

```
@@ -11,9 +11,7 @@ jobs:
       with:
         submodules: recursive
     - name: install tools
-      run: |
-        gem install --user-install cbor-diag
-        gem install --user-install cddl -v 0.10.3
+      run: gem install --user-install cbor-diag cddl
```

This puts it back to its original form which wouldn't catch the case where 0.10.6 wasn't installed.
```suggestion
      run: |
        gem install --user-install cbor-diag
        gem install --user-install cddl -v 0.10.6
```

---

this CI workflow will be invoked from now onwards.  Since Carsten has released 0.10.6 this will always pull at least 0.10.6.  This is inline with the local setting reflected in the Gemfile (i.e., `>=0.10.6`).